### DISCUSSION TITLE
Capacitor Android Implementation

### DISCUSSION THREAD

**Discussion by gitsunny404** [abacritt/angularx-social-login#694] [Q&A]
I am new to angular and all.

Can I use it on android with capacitorjs.

I have tried for Google Auth, It is working in angular but after building android app via capacitor. It is showing me white screen.

Also, I want the pop menu for login and not the redirect to any browser in android and then come back to app.

Thank you.

**Comment by stevenwaskey** [comment]
Hey @gitsunny404 
Did you find any solutions here?
